syne_tune/experiments.py

```diff
@@ -61,30 +64,87 @@ def creation_date(self):
         """
         return datetime.fromtimestamp(self.metadata[ST_TUNER_CREATION_TIMESTAMP])
 
-    def plot(self, **plt_kwargs):
+    def plot_hypervolume(
+        self, reference_points: np.ndarray = None, figure_path: str = None, **plt_kwargs
+    ):
+        """Plot best hyervolume value as function of wallclock time
+
+        :param reference_points: Reference points for hypervolume calculations.
```

Comment:
nit: If `:param:` in docstring is multi-line, just use one tab for 2-last line

Reply:
OK makes sense